src/libcadet/model/binding/StericMassActionBinding.cpp

```diff
@@ -329,6 +330,34 @@ namespace binding
 	}
 }  // namespace binding
 
+class NoJacobianStericMassActionBinding : public model::StericMassActionBinding {
+
+public:
+
+	static const char* identity() { return "NO_JACOBIAN_STERIC_MASS_ACTION"; }
+
+	CADET_BINDINGMODELBASE_BOILERPLATE
+
+protected:
+
+	const char* name() const CADET_NOEXCEPT override { return identity(); }
+	bool implementsAnalyticJacobian() const CADET_NOEXCEPT override { return false; }
+
+	template <typename RowIterator>
+	void jacobianImpl(double t, unsigned int secIdx, const ColumnPosition& colPos, double const* y, double const* yCp, int offsetCp, RowIterator jac, LinearBufferAllocator workSpace) const
+	{
+		BindingModelBase::jacobianImpl(t, secIdx, colPos, y, yCp, offsetCp, jac, workSpace);
```

Comment:
I made sure that this part of the code is actually executed by jumping to this place (and one level deeper) in the debugger. I had to add the Boilerplate for the ``NoJacobianStericMassActionBinding`` class, otherwise the code from ``StericMassActionBinding `` would be executed.